packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -611,6 +611,33 @@ function pickStyleKeys( treeToPickFrom ) {
 	return Object.fromEntries( clonedEntries );
 }
 
+function scopeFeatureSelectors( scope, selectors ) {
```

Comment:
Blocks can define their own custom selectors for features such as color, typography, border etc. When such a block is styled as an inner block to a block style variation, its custom feature selectors need to be scoped by the block style variations selector to still be correctly applied. This function helps with that.